torch_lr_finder/lr_finder.py

```diff
@@ -430,6 +431,8 @@ def plot(self, skip_start=10, skip_end=5, log_lr=True, show_lr=None, ax=None):
                 matplotlib axes object and the figure is not be shown. If `None`, then
                 the figure and axes object are created in this method and the figure is
                 shown . Default: None.
+            suggestion (bool, optional): True to plot the point with the steepest gardient,
```

Comment:
Typo gardient -> gradient.

(also, don't forget to update the parameter name if you change it)

Reply:
```suggestion
            suggest_lr (str , optional): suggest a learning rate
                - 'steepest': the point with steepest gradient (minimal gradient)
```

Reply:
I would actually suggest keeping `suggest_lr` as a `bool` since there's only one algorithm for suggesting a learning rate. I don't know when or if more will be implemented and in the meantime having it as an `str` might be confusing for users if there's only one choice.